match/lib/match/module.rb

```diff
@@ -4,12 +4,16 @@ module Match
   Helper = FastlaneCore::Helper # you gotta love Ruby: Helper.* should use the Helper class contained in FastlaneCore
   UI = FastlaneCore::UI
   ROOT = Pathname.new(File.expand_path('../../..', __FILE__))
-  DESCRIPTION = "Easily sync your certificates and profiles across your team using git"
+  DESCRIPTION = "Easily sync your certificates and profiles across your team"
 
   def self.environments
     return %w(appstore adhoc development enterprise)
   end
 
+  def self.storage_modes
+    return %w(git google_cloud)
```

Comment:
`google_cloud` isn't implemented yet so we should probably remove this for now 🙃